encoders/image/OnnxImageEncoder/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM jinaai/jina:devel
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN apt-get update && apt-get install --no-install-recommends -y git curl && \
+    pip install -r requirements.txt
+
+# RUN pip install -r requirements.txt
+COPY . /
+
+# for downloading pre-trained model
+RUN bash download.sh
+
+# for testing the image
+RUN pip install pytest && pytest -v
```

Comment:
I would remove `RUN bash download.sh` and I would do:

`RUN download.sh && pip install pytest && pytest -v -s && rm -rf mobilenetv2-1.0`

so that we do not upload an image with a model in it, since at running time one might use a different one or can map a `volume`

Reply:
Yes sure, updated the Dockerfile with the changes

Reply:
Does the `download.sh` work for you? In my case, it ends up with an empty file.